File: cs50 projects/sentimental-mario-less/mario.py
from cs50 import get_int

h = get_int("Height: ")

while ((h < 1) or (h > 8)):
    h = get_int("Height: ")

for r in range(h):
    for c in range(h):
        if c + r >= h - 1:
            print("#", end="")
        else:
            print(" ", end="")
    print()

File: 68290667-main/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
int main(void)
{
    //ask for height of pyramid
    int height;
    do{
        height = get_int("Height: \n");
    }
    while ((height <1) || (height >8));

    for(int row =0; row<height; row++) {
        int col =  0;
        for(col =0;col < height-(row+1); col++) {
            printf("%s", " ");
        }
        for(;col<height;col++) {
            printf("%s", "#");
        }
        printf("%s", "\n");

    }
}